modules/virt-creating-a-custom-image-boot-source-web.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
+
+[id="virt-creating-a-custom-disk-image-boot-source-web_{context}"]
+= Creating a custom boot source
+
+Sometimes you want to prepare a custom disk image, based on an existing disk image, for use as a boot source.
+
+This procedure guides you through:
+
+* Preparing a custom disk image.
+* Creating a boot source from the custom disk image.
+* Attaching the boot source to a custom template.
+
+.Procedure
+. In the {VirtProductName} console, click *Workloads* > *Virtualization* from the side menu.
+. Click the *Templates* tab.
+. Click the link in the *Source provider* column for the template you want to customize. A window displays, indicating that the template currently has a *defined* _source_.
```

Comment:
Suggest dropping the emphasis on `_source_`, unless there's a meaning i'm not getting.

Reply:
The source is technically a variable, but I find it confusing too, so I'm dropping it. It's not really helping anything.